### DISCUSSION TITLE
Can't lauch lazer since last update

### DISCUSSION THREAD

**Discussion by Figaros** [ppy/osu#23326] [Ideas] [answered]
Since last update disabled vulkan, i think that is the problem.
However, i want to launch lazer but i don't know how to change to opengl or directx outside the game itself.
Does anyone know how to do it?

**Comment by monochrome22** [comment, marked as answer]
Try this: https://github.com/ppy/osu/discussions/23065#discussioncomment-5481552

"To make the game work again you can either delete framework.ini from the game files, or if feeling particularly attached to your settings, remove the Renderer = Vulkan line from that file."

**Reply by Figaros** [reply]
I can't find a file named framework.ini , the only files i find were .dll , like osu.framework.dll and don't don't if it's a good idea deleting them

**Reply by monochrome22** [reply]
It should be in Appdata/Roaming/osu

**Reply by Figaros** [reply]
oh thank you! solved!
